### ISSUE TITLE
请问作者考虑新版本加个开机自启或者定时启动设定吗？？？已经添加软件目录到环境变量还是会出现各种不稳定现象 

### ISSUE THREAD

**Issue by douying2334** [ihmily/DouyinLiveRecorder#343]
添加环境变量就不会提示缺失ffmpeg  ,但是又出现新的问题 就是会提示 

.....................................................
系统代理检测中，请耐心等待...
INFO：未检测到全局/规则网络代理，请检查代理配置（若无需录制海外直播请忽略此条提示）
请输入要录制的主播直播间网址（尽量使用PC网页端的直播间地址）: 

只有手动打开才会检测到直播地址  自动打开就提示检测不到直播地址  已经添加软件目录到环境变量

现在不得不手动打开了 就是电脑有时候重启忘记打开软件就会错过 录制

**Comment by ihmily** [comment]
```
请输入要录制的主播直播间网址（尽量使用PC网页端的直播间地址）:
```
出现这个是因为找不到 URL_config.ini文件或者里面内容为空，没有直播间地址

**Comment by douying2334** [comment]
> ```
> 请输入要录制的主播直播间网址（尽量使用PC网页端的直播间地址）:
> ```
> 
> 出现这个是因为找不到 URL_config.ini文件或者里面内容为空，没有直播间地址

是的 但是 手动打开就会检测到直播地址    而且确认了  URL_config.ini文件有直播间地址，估计是啥原因找不到，尝试将软件目录 软件config目录都添加环境变量还是不行。。 所以只能希望软件有没有可能设定个自启按钮，

看更新日志还没修复花屏问题，作者加油！！用了五六款录制软件 ，作者的这款是最稳定最好用的了 ，作者大大真是优秀的人才，还免费分享 大好人

**Comment by douying2334** [comment]
> ```
> 请输入要录制的主播直播间网址（尽量使用PC网页端的直播间地址）:
> ```
> 
> 出现这个是因为找不到 URL_config.ini文件或者里面内容为空，没有直播间地址

目前用的是梦畅闹钟有个定时启动软件的功能

**Comment by douying2334** [comment]
> 尝试将软件目录 软件config目录都添加环境变量还是不行。

您好！刚才有人说提交了代码 好像可以修复说 https://github.com/ihmily/DouyinLiveRecorder/issues/257#issuecomment-2098514603

现在非常麻烦 每天要惦记晚上记得手动打开录制程序 都每天晚上定闹钟提醒了， 其实我电脑是不关的， 只是怕运行这个录制玩网游有 风险呀 会 被当成挂 ，只能玩游戏时关掉。 主播晚上一般开播就晚上记得打开。  所以想想开机自启不好 还是定时晚上启动好点，毕竟方便

**Comment by legel840100558** [comment]
你玩的不会是warframe吧 我后台开着玩没事

**Comment by douying2334** [comment]
> 你玩的不会是warframe吧 我后台开着玩没事

彩虹六号 ，cf ,cs2.  哦
